modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.h

```diff
@@ -92,22 +92,22 @@ class LBSSolver : public opensn::Solver
   const std::vector<LBSGroupset>& Groupsets() const;
 
   /// Adds a point source to the solver.
-  void AddPointSource(PointSource&& point_source);
```

Comment:
apologies for cluttering the discussion, but what is `&&` again? [i know it's gone now]